### ISSUE TITLE
Flux model file location

### ISSUE THREAD

**Issue by weatherman6996** [cocktailpeanut/fluxgym#147]
I've already downloaded flux model. Can I put it somewhere to skip base model dowlaoding ?

**Comment by KeeeeperOfTheLight** [comment]
same here

**Comment by metamountain** [comment]
Same here.

**Comment by KeeeeperOfTheLight** [comment]
> You can easily add more to the supported models list by editing the [models.yaml](https://github.com/cocktailpeanut/fluxgym/blob/main/models.yaml) file.

yeah dude i saw that, i can modify models location in comfyui cuz it so easy.
 how to write it in this yaml file.

**Comment by dekahobby** [comment]
how to use this model btw? https://huggingface.co/ashen0209/Flux-Dev2Pro

**Comment by KeeeeperOfTheLight** [comment]
> how to use this model btw? https://huggingface.co/ashen0209/Flux-Dev2Pro

don't know dude sorry i searched about it and i found nothing.
 only the model.

**Comment by Vigilence** [comment]
Latest update to fluxgym includes this model.

![Screenshot 2024-10-07 145613](https://github.com/user-attachments/assets/e200da8f-bee1-4b0f-aa6c-5c7b3f2ea479)
